### PR TITLE
Invalid transction link fix

### PR DESCRIPTION
**Pull request by claudiaHash**
Fixes # 1033

Changes proposed in this PR:

- get subgraph nftUpdates query fix

### CODE REVIEW

**Comment by mihaisc** [comment]
We can also update the text based on the type of update 
image

File: src/components/Asset/AssetContent/EditHistory.tsx [hunk 1]
```diff
@@ -10,20 +10,38 @@ import styles from './EditHistory.module.css'
...
 
+  function getUpdateType(type: string): string {
+    switch (type) {
+      case 'METADATA_CREATED':
+        return 'published'
+      case 'METADATA_UPDATED':
+        return 'updated'
+      case 'STATE_UPDATED':
+        return 'state updated'
+      case 'TOKEN_UPDATED':
```
Comment (mihaisc) [inline src/components/Asset/AssetContent/EditHistory.tsx:39]: this is `TOKENURI_UPDATED`, not token. It's when `tokenUri` is updated on the nft

File: src/components/Asset/AssetContent/EditHistory.tsx [hunk 2]
```diff
@@ -10,20 +10,38 @@ import styles from './EditHistory.module.css'
...
+  function getUpdateType(type: string): string {
+    switch (type) {
+      case 'METADATA_CREATED':
+        return 'published'
+      case 'METADATA_UPDATED':
+        return 'updated'
+      case 'STATE_UPDATED':
+        return 'state updated'
+      case 'TOKENURI_UPDATED':
+        return 'token uri updated'
```
Comment (mihaisc) [inline src/components/Asset/AssetContent/EditHistory.tsx:40]: not sure if people would properly understand this @kremalicious any ideas how the label should look ? `nft metadata updated` and for `METADATA_UPDATED` have `asset metadata updated` ?

Comment (kremalicious) [reply src/components/Asset/AssetContent/EditHistory.tsx:40]: yup, think "NFT metadata updated" for this case is better, for asset updated we can leave as is with "updated"

**Review by kremalicious — APPROVED** [review]

**Comment by kremalicious** [comment]
we can ignore Vercel, just had to switch it to Gatbsy to get a v3 deployment out, and now set it back to Next.js builds.

**Pull request merged**